Automattic/wp-calypso
# A4A: Sites Dashboard - Update Dataviews version to latest

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/311

## Proposed Changes

* This PR updates wordpress/dataviews to the latest version (v1.0.0).
  * List of issues fixed:
    * Fixed status filter behavior, which changed in this version
    * Fixed dataviews styling when preview pane is open
    * Fixed scrolling 

## Testing Instructions

* One of our components changed, so run `yarn install` to make sure everything is set-up properly.
* Run `yarn start-a8c-for-agencies`
* Go to http://agencies.localhost:3000/sites
* Verify that the new version of our dashboard's dataview component is functioning properly in A4A. There have been some changes compared to our current version, which are outlined below, along with one known issue that we must disregard during testing.

Changes:

**Filter selection**

Previous Version:
![image](https://github.com/Automattic/wp-calypso/assets/37049295/669ffe50-6ada-4937-bcfb-8e693df3eafd)

Latest:

![image](https://github.com/Automattic/wp-calypso/assets/37049295/f15486ff-7607-4b92-ad26-c1c9561a2456)

![image](https://github.com/Automattic/wp-calypso/assets/37049295/67d621ff-5fd8-4899-bc69-9498d3a4a2bf)


**When Preview Pane is open, filters now show multiple lines**

Previews Version:
![image](https://github.com/Automattic/wp-calypso/assets/37049295/b78fcc82-71c0-49f8-ae35-2b994f9d9c25)

Latest:
![image](https://github.com/Automattic/wp-calypso/assets/37049295/fdbd9e6b-08a3-439d-a21e-de1568f58ceb)


**Known Issue - Please ignore this one for now**

When the Preview Pane is open and pagination is visible, the page's alignment looks off.

![image](https://github.com/Automattic/wp-calypso/assets/37049295/14d979c3-155d-46f7-8df7-25c612a990cb)


## Pre-merge Checklist
